src/modules/form/fheo.tsx

```diff
@@ -31,6 +31,12 @@ export const Fheo = () => {
                                 'previous': 'GO BACK',
                                 'submit': 'SUBMIT MY COMPLAINT'
                             }
+                        },
+                        buttonSettings: {
+                            showPrevious: true,
+                            showNext: true,
+                            showCancel: false,
+                            showSubmit: true
                         }
```

Comment:
this setting can be done in formio since, we want less formio code in the react app

Reply:
<img width="266" alt="Screen Shot 2021-02-17 at 8 37 10 AM" src="https://user-images.githubusercontent.com/69588255/108212010-65aef880-70fb-11eb-82b0-2447e1a03f00.png">

Reply:
Unfortunately, no, we can't set this in form.io if we want it to work.  The only way the button navigation works is if we do NOT set this setting on each individual panel.  Apparently the panel settings override the settings if we try to set them with code (like when the edit button is clicked on the review page).  So, we have to remove ALL the panel navigation button settings in form.io from every panel.  And the only way to have the "cancel" button removed on form display is to do what you see above.

I know what you are saying about removing code from the react app in favor of inline embedding.  We *should* be able to set both of the above options (buttonSettings and i18n) using inline embedding, so I am not too worried about this being something we could not recreate in an inline embed scenario.

Reply:
removing the panel navigation was the recommendation from form.io